satpy/composites/config_loader.py

```diff
@@ -131,9 +131,12 @@ def _get_modifier_loader_from_config(modifier_name, modifier_info):
             loader = modifier_info.pop('modifier', None)
             if loader is None:
                 loader = modifier_info.pop('compositor')
-                warnings.warn("Modifier '{}' uses deprecated 'compositor' "
-                              "key to point to Python class, replace "
-                              "with 'modifier'.".format(modifier_name))
+                warnings.warn(
+                    "Modifier '{}' uses deprecated 'compositor' "
+                    "key to point to Python class, replace "
+                    "with 'modifier'.".format(modifier_name),
+                    stacklevel=2
```

Comment:
This should probably be at least stacklevel 3 shouldn't it? This is a helper class inside the config loading so we probably want the caller of the caller of this method? Oh actually this method is called from the `_load_config_modifier` method in this same class and that one from another method so maybe stacklevel 5?